build_runner/test/generate/build_integration_test.dart

```diff
@@ -60,10 +60,8 @@ main(List<String> args) async {
         var result = await runDart('a', 'tool/build.dart',
             args: ['build', '--delete-conflicting-outputs']);
         expect(result.exitCode, 0, reason: result.stderr as String);
-        expect(
-            result.stdout,
-            contains('BuildDefinition: Invalidating asset graph due to '
```

Comment:
are these names no longer present or you just wanted to simplify the tests? I think they do provide some value personally (the names being there, I am ambivalent about the tests).

Reply:
They are no longer present. I felt like they mainly added value for us. The end user probably didn't care about the particulars ways we split up functionality in this package or the labels we gave them.

Reply:
It gave you some indication of where the log was coming from but its probably true that for most users that didn't mean much.... I am ok with removing them.